src/controllers/__tests__/products.test.js

```diff
@@ -94,6 +94,7 @@ describe('Products routes', () => {
         type: 'Product',
         location: 'Ankara',
         condition: 'Used',
```

Comment:
`location` field of `Product` model has been modified. Please check `Product` model and update `location` field here accordingly.